util/genrest/protomodelcreator.go

```diff
@@ -264,15 +266,29 @@ func getMixinsForAPI(mixinRules indexedRules, api string) Mixins {
 	return files
 }
 
+func getMixinFiles() (files []*descriptor.FileDescriptorProto) {
+	for _, descriptors := range mixinDescriptors {
+		files = append(files, descriptors...)
+	}
+	return
+}
+
 // mixinDescriptors maps fully qualified proto service names of mixins implemented by Showcase to a
 // list of FileDescriptors containing the definitions needed for that service.
 var mixinDescriptors map[string][]*descriptor.FileDescriptorProto
 
 func init() {
-	// TODO(noahdietz): Figure out how to add Locations and IAMPolicy to this list.
 	mixinDescriptors = map[string][]*descriptor.FileDescriptorProto{
 		"google.longrunning.Operations": {
-			protodesc.ToFileDescriptorProto(longrunning.File_google_longrunning_operations_proto),
+			protodesc.ToFileDescriptorProto(longrunningpb.File_google_longrunning_operations_proto),
```

Comment:
In our Go proto world, we moved all generated code that has a gapic to a submodfule of https://pkg.go.dev/cloud.google.com/go, including the protobuf/grpc generated stubs so they can be versioned as one module, instead of in the separate google.golang.org/genproto (which has no semantic versioning). Locations doesn't have a GAPIC (but IAM and LRO do for "reasons"), so Locations remains in google.golang.org/genproto, while IAM was to cloud.google.com/go/iam/apiv1/iampb, and same for LRO.